apps/settings.py.tmpl

```diff
@@ -2,31 +2,38 @@ RUN_MODE = 'Live'
 
 SETTINGS_NAME = 'Eval configuration'
 
-DEFAULT_CERT_FILE = 'some.crt'
-DEFAULT_KEY_FILE = 'some.key'
-DEFAULT_CA_FILE = 'some-cacerts.cert'
+DEFAULT_CERT_FILE = '/usr/local/ssl/certs/some.cert'
+DEFAULT_KEY_FILE = '/usr/local/ssl/certs/some.key'
+DEFAULT_CA_FILE = '/usr/local/ssl/certs/some-cacerts.cert'
 
 FILTER_FILE = 'data/PersonRegGroupDefs.xml'
 
 GWS_CONF = {
-    'HOST':  'base-url',
+    'HOST':  'https://dev.groups.uw.edu',
     'SOCKET_TIMEOUT':  900.0,
-    'CERT_FILE': 'some.crt'
+    'CERT_FILE': 'some.crt',
     'KEY_FILE': 'some.key'
 }
 
 IRWS_CONF = {
-    'HOST':  'base-url',
-    'SERVICE_NAME': 'registry-xxx',
+    'HOST':  'https://mango-environment.u.washington.edu:646',
+    'SERVICE_NAME': 'registry-environment',
     'SOCKET_TIMEOUT':  65.0,
 }
 
 PAC_CONF = {
-    'SMTP_SERVER': 'some-smtp-server',
+    'SMTP_SERVER': 'appsubmit.cac.washington.edu',
+    'SMTP_USE_SSL': False, # TLS or SSL, not both
+    'SMTP_USE_TLS': True,  # TLS or SSL, not both
```

Comment:
Since these are mutually exclusive, would it make more sense to to use an enum or other structure of a "list of choices"?

```python
from enum import Enum

class SMTPConnectionType(Enum):
    tls = 'tls'
    ssl = 'ssl'

# When reading in the settings, you can just use a string value, 
# which will error if the provided value doesn't match one of the choices
def load_settings():
    # . . . blah blah . . . 
    smtp_conn_type = SMTPConnectionType(conf['SMTP_CONN_TYPE'])    
```

And the settings:

```python

PAC_CONF = {
    # . . . 
    'SMTP_CONN_TYPE': 'tls'
}
```
...granted, this may be more of a refactor than you're willing to do, but I thought I'd present the thought experiment, anyway :)

Reply:
I'd prefer this but I think I need to move on from this bugfix for now.